docs/reference/python-sdk.rst

```diff
@@ -33,9 +33,9 @@ The next step is to call create_experiment():
    exp = client.create_experiment(config="my_config.yaml", model_dir=".")
    print(f"started experiment {exp.id}")
 
-The returned object will be an ``ExperimentReference`` object, which has methods for controlling the
-lifetime of the experiment running on the cluster. In this example, we will just wait for the
-experiment to complete.
+The returned object will be an ``Experiment`` object, which has methods for controlling the lifetime
+of the experiment running on the cluster. In this example, we will just wait for the experiment to
+complete.
```

Comment:
The returned object is an ``Experiment`` object, which offers methods to manage the experiment's lifecycle. In the folowing example, we simply await the experiment's completion.